src/roman_numerals/core.clj

```diff
@@ -1,7 +1,12 @@
 (ns roman-numerals.core)
 
-(defn convert
-  "I convert arabic to roman numerals"
-  [x]
-  (if (= x 0)
-    ""))
+(defn to-numerals [arabic numerals] 
+  (cond (= arabic 0) numerals
+        (<= arabic 3) (to-numerals (- arabic 1) (str numerals "I"))
```

Comment:
Can you find a way to remove this duplication? There's a pattern here that looks something like below. How would a test for 20 cause your code to change? It would be ideal if your code didn't have to change.

(<= arabic <number>) (to -numerals (- arabic <number>) (str numerals <roman>))